optuna/pruners/hyperband.py

```diff
@@ -70,20 +78,42 @@ class HyperbandPruner(BasePruner):
     """
 
     def __init__(
-        self, min_resource=1, reduction_factor=3, n_brackets=4, min_early_stopping_rate_low=0
-    ):
-        # type: (int, int, int, int) -> None
-
+        self,
+        max_resource: int = 100,
```

Comment:
Could you briefly motivate the choice of 100?

Reply:
Sorry for a mistake. I should make it not 100 but 80, because the previous default `n_brackets` is 4 and it satisfies `4 = n_brackets = \log_{reduction_factor}(max_resource) + 1 = \log_3(80) + 1`.